### PR TITLE
adding multiple rules to single backend

### PR DESCRIPTION
**Pull request by jyotsnaravikumar**
## Purpose
Document github issues filed for multiple routing rules

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone https://github.com/Azure-Samples/frontdoor-appservice-vnet-terraform/
cd frontdoor-appservice-vnet-terraform

```

* Test the code

```hcl
terraform init
terraform apply -var-file="terraform.tfvars"
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,8 +33,6 @@ terraform apply -var-file="terraform.tfvars"
```
Comment (edaena) [inline README.md:48]: Can you add the name of the project at the top of this file too. Right now it only says "Project Name"

**Review by yradsmikham — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jyotsnaravikumar) [reply README.md:48]: Yes, good point @edaena, done! Update description as well.

File: main.tf [hunk 2]
```diff
@@ -11,6 +11,74 @@ resource "azurerm_resource_group" "rg" {
   location = var.rg_location
 }
 
+resource "azurerm_app_service_plan" "asp" {
+  name                = "rocket-asp"
```
Comment (bnookala) [inline main.tf]: probably change the name to be configurable , or remove internal team name references

File: main.tf [hunk 3]
```diff
@@ -11,6 +11,74 @@ resource "azurerm_resource_group" "rg" {
...
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Standard"
```
Comment (bnookala) [inline main.tf]: can the tier be configurable? i remember we had issues with certain tiers

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Project Name
+# Frontdoor terraform script with App Service
```
Comment (edaena) [inline README.md]: nit: The official name is "Azure Front Door" might want to separate Frontdoor into two words.

File: main.tf [hunk 5]
```diff
@@ -11,6 +11,74 @@ resource "azurerm_resource_group" "rg" {
...
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  app_service_plan_id = azurerm_app_service_plan.asp.id
+
+  site_config {   
+    /* Provision voting App from https://github.com/Azure-Samples/azure-voting-app-redis */
+    linux_fx_version = "COMPOSE|${filebase64("docker-compose.yaml")}"
+    /* App Service access limited to FrontDoor Only */
+
+    ip_restriction = [
```
Comment (bnookala) [inline main.tf:42]: add a comment about these being internal azure IPs

File: main.tf [hunk 6]
```diff
@@ -19,23 +87,6 @@ resource "azurerm_frontdoor_firewall_policy" "demowafpolicy" {
   custom_block_response_status_code = 403
   custom_block_response_body        = "YmxvY2tlZCBieSBmcm9udGRvb3I="
```
Comment (bnookala) [inline main.tf:95]: probably add a comment that states this is a base 64 encoded string "blocked by frontdoor"

File: main.tf [hunk 7]
```diff
@@ -78,10 +129,10 @@ resource "azurerm_frontdoor" "rocketdemofd" {
   }
 
   backend_pool {
-    name = "rocketDemoBackendBing"
+    name = "rocketDemoBackendVoting"
     backend {
-      host_header = "www.bing.com"
-      address     = "www.bing.com"
+      host_header = "rocket-appservice.azurewebsites.net"
```
Comment (bnookala) [inline main.tf]: this should be configured based on the name of the app service that the user provides

File: main.tf [hunk 8]
```diff
@@ -11,6 +11,74 @@ resource "azurerm_resource_group" "rg" {
...
+    tier = "Standard"
+    size = "S1"
+  }
+}
+
+/* Provision existing voting App from https://github.com/Azure-Samples/azure-voting-app-redis */
+/* Limit access to the App Service from FrontDoor Only */
+
+resource "azurerm_app_service" "apsvc" {
+  name                = "rocket-appservice"
```
Comment (bnookala) [inline main.tf]: make this a user configurable var

File: README.md (hunk 4 above)
Comment (bnookala) [inline README.md]: Suggest changing this to something like: Using Terraform to deploy and configure Azure Front Door with an Azure App Service

File: README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
 
-This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config exposed on a private vnet
+This project provides a end to end terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config. The script provisions an existing voting App from https://github.com/Azure-Samples/azure-voting-app-redis and limits access to this backend from Azure Frontdoor.
```
Comment (bnookala) [inline README.md]: rewrote this a bit:

This project end to end terraform scripts for provisioning a WAF enabled Azure Front Door with backend pools set with an existing Azure App service, routing rules with caching config. 

These scripts:
1. Provision an Azure Front Door with Web Application Firewall (WAF) enabled
2. Provision a sample application on Azure App Service(Azure Vote - https://github.com/Azure-Samples/azure-voting-app-redis)
3. Configures Front Door to route traffic to the App Service with caching configuration
4. Places limits on inbound traffic to the App Service to be limited to Azure Infrastructure

**Review by bnookala — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (jyotsnaravikumar) [reply README.md]: Updated to Bhargav's input from below

File: main.tf (hunk 2 above)
Comment (jyotsnaravikumar) [reply main.tf]: Done!

File: main.tf (hunk 5 above)
Comment (jyotsnaravikumar) [reply main.tf:42]: Done!

File: main.tf (hunk 7 above)
Comment (jyotsnaravikumar) [reply main.tf]: Done!

File: README.md (hunk 4 above)
Comment (jyotsnaravikumar) [reply README.md]: Done!

File: README.md (hunk 9 above)
Comment (jyotsnaravikumar) [reply README.md]: DOne!

**Review by edaena — APPROVED** [review]

File: main.tf (hunk 3 above)
Comment (jyotsnaravikumar) [reply main.tf]: Done!

File: main.tf (hunk 6 above)
Comment (jyotsnaravikumar) [reply main.tf:95]: Done

File: main.tf (hunk 8 above)
Comment (jyotsnaravikumar) [reply main.tf]: Done

File: README.md [hunk 10]
```diff
@@ -1,6 +1,13 @@
-# Project Name
+# Using Terraform to deploy and configure Azure Front Door with an Azure App Service
 
-This project provides a sample terraform script for provisioning a WAF enabled frontdoor with backend pools set with an existing Azure App service, routing rules with  caching config exposed on a private vnet
+This project end to end terraform scripts for provisioning a WAF enabled Azure Front Door with backend pools set with an existing Azure App service, routing rules with caching config.
```
Comment (bnookala) [inline README.md]: err… apologies, the wording here is incomplete!! Should be: "This project provides end to end…"

File: README.md [hunk 11]
```diff
@@ -1,6 +1,13 @@
...
+
+These scripts:
+
+Provision an Azure Front Door with Web Application Firewall (WAF) enabled
```
Comment (bnookala) [inline README.md]: bullet points for these or number them

**Review by bnookala — CHANGES_REQUESTED** [review]
last two things, and then good to go

**Review by bnookala — APPROVED** [review]

File: README.md (hunk 10 above)
Comment (jyotsnaravikumar) [reply README.md]: DOne! I had end to end wordings before and then removed them :) and have it back :)

File: README.md (hunk 11 above)
Comment (jyotsnaravikumar) [reply README.md]: done

**Pull request merged**